src/Entity/Item.php

```diff
@@ -15,94 +15,80 @@ class Item
     private int $id;
 
     #[ORM\Column]
-    private int $quatidade;
+    private int $quantidade;
 
-    /**
-     * @var Collection<int, Produto>
-     */
-    #[ORM\OneToMany(targetEntity: Produto::class, mappedBy: 'item')]
-    private Collection $produtoId;
-
-    /**
-     * @var Collection<int, Carrinho>
-     */
-    #[ORM\ManyToMany(targetEntity: Carrinho::class, inversedBy: 'items')]
-    private Collection $carrinhoId;
-
-    public function __construct()
-    {
-        $this->produtoId = new ArrayCollection();
-        $this->carrinhoId = new ArrayCollection();
+    #[ORM\Column]
+    private int $valor;
+
+    #[ORM\ManyToOne]
+    private Produto $produto;
+
+    #[ORM\ManyToOne]
+    private Carrinho $carrinho;
+    public function __construct(Produto $produto, Carrinho $carrinho, int $quantidade, int $valor)
```

Comment:
tem um espaço a menos, e quando tiver mais de 3 parametros é legal quebrar a linha do construtor ou dos métodos também...

```suggestion
    #[ORM\ManyToOne]
    private Carrinho $carrinho;

    public function __construct(
        Produto $produto,
        Carrinho $carrinho,
        int $quantidade,
        int $valor
    )
```